### PR TITLE
Fix character counting problems with utf8	

### PR DESCRIPTION
**Pull request by mingrammer**
1. Use 'utf8' only for counting characters in string
2. Modify `sample.md` had wrong count numbers.
3. Restore test

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -24,6 +24,9 @@ def trans_coverage_file(file, ext=None):
         with open(file, "r") as myfile:
             s = myfile.read()
 
+            if type(s) == bytes:
+                s = s.decode('utf8')
```
Comment (hunkim) [inline main.py:28]: Is it always safe? Should we add try around this statement?

**Comment by mingrammer** [comment]
Hm, which exception?

File: main.py (hunk 1 above)
Comment (hunkim) [reply main.py:28]: Like UnicodeDecodeError exception?

Comment (mingrammer) [reply main.py:28]: Uh, I agree! thanks for catching

Comment (mingrammer) [reply main.py:28]: If add 'utf8' encoding when open file, that exception might not be raised i think. right?

**Comment by hunkim** [comment]
just add try: catch (do nothing) to make it safe? I have experienced this
exception with open.

File: main.py (hunk 1 above)
Comment (hunkim) [reply main.py:28]: perhaps, just add try: catch (do nothing) to make it safe? I have experienced this
exception with open.

**Comment by mingrammer** [comment]
It is better to print some error messages to console when exception rised.

**Comment by mingrammer** [comment]
@hunkim Please add try:catch statement you mentioned. 
I can not work on today :(

**Comment by hunkim** [comment]
Sure. However, in the current method the translate rate is too low.

For example, translation is completed, but we say only 28%, since it has a
long source code:
https://github.com/tensorflowkorea/tensorflow-kr/blob/master/g3doc/get_started/index.md

Should we `others *= 2` for now? Do you know how to extract only text
(excluding source code)?

**Comment by mingrammer** [comment]
@hunkim this issue is not easy to determine right now. I think we can detect the some text is whether source code or not.

**Comment by hunkim** [comment]
Then, for now, how about #18?

**Comment by mingrammer** [comment]
If for just now, I'm ok. 
I'll improve the rates later. :)

**Pull request merged**